content/topics/clean-code/python/_index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Clean Code for Python
+ready: True
+---
+
```

Comment:
Another best practice is to try to avoid the use of legacy python (Python <=2.7 ). If you are starting a new project it's always wise to use the latest and greatest except if it is incompatable with other parts of your project.

Eg: even though 3.8 is out and we are all very excited about the new walrus, google app engine only supports up to 3.7.